style/emoji/LICENSE

```diff
@@ -0,0 +1,3 @@
+Copyright 2014 Twitter, Inc and other contributors
```

Comment:
Hmm, I'm not sure how best to represent this, but I modified the language in https://github.com/andreasgal/j2me.js/commit/d2f7aad7c04c6140d5ada011845ecf0801d66d95 to say:

> Adapted from graphics copyright 2014 Twitter, Inc and other contributors

How does that sound?